test/combine.jl

```diff
@@ -43,30 +43,17 @@ end
     ba   = tail(BA)
 
     @testset "takes colnames kwarg correctly" begin
-        @test merge(cl, ohlc["High", "Low"], colnames=["a","b","c"]).colnames == ["a", "b", "c"]
-        @test merge(cl, op, colnames=["a","b"]).colnames                      == ["a", "b"]
-        @test_throws ErrorException merge(cl, op, colnames=["a"])
-        @test_throws ErrorException merge(cl, op, colnames=["a","b","c"])
-
-        @test merge(cl, ohlc["High", "Low"], :inner, colnames=["a","b","c"]).colnames == ["a", "b", "c"]
-        @test merge(cl, op, :inner, colnames=["a","b"]).colnames == ["a", "b"]
-        @test_throws ErrorException merge(cl, op, :inner, colnames=["a"])
-        @test_throws ErrorException merge(cl, op, :inner, colnames=["a","b","c"])
-
-        @test merge(cl, ohlc["High", "Low"], :left, colnames=["a","b","c"]).colnames == ["a", "b", "c"]
-        @test merge(cl, op, :left, colnames=["a","b"]).colnames          == ["a", "b"]
-        @test_throws ErrorException merge(cl, op, :left, colnames=["a"])
-        @test_throws ErrorException merge(cl, op, :left, colnames=["a","b","c"])
-
-        @test merge(cl, ohlc["High", "Low"], :right, colnames=["a","b","c"]).colnames == ["a", "b", "c"]
-        @test merge(cl, op, :right, colnames=["a","b"]).colnames == ["a", "b"]
-        @test_throws ErrorException merge(cl, op, :right, colnames=["a"])
-        @test_throws ErrorException merge(cl, op, :right, colnames=["a","b","c"])
-
-        @test merge(cl, ohlc["High", "Low"], :outer, colnames=["a","b","c"]).colnames == ["a", "b", "c"]
-        @test merge(cl, op, :outer, colnames=["a","b"]).colnames == ["a", "b"]
-        @test_throws ErrorException merge(cl, op, :outer, colnames=["a"])
-        @test_throws ErrorException merge(cl, op, :outer, colnames=["a","b","c"])
+        @test merge(cl, ohlc[:High, :Low], colnames=[:a, :b, :c]).colnames == [:a, :b, :c]
+        @test merge(cl, op, colnames=[:a, :b]).colnames                    == [:a, :b]
+        @test_throws ErrorException merge(cl, op, colnames=[:a])
+        @test_throws ErrorException merge(cl, op, colnames=[:a, :b, :c])
+
+        for mode ∈ [:inner, :left, :right, :outer]
+          @test merge(cl, ohlc[:High, :Low], mode, colnames=[:a, :b, :c]).colnames == [:a, :b, :c]
```

Comment:
Should be the Julia standard 4 spaces for indentation

Reply:
fixed